torchtext/data/field.py

```diff
@@ -81,10 +81,9 @@ def preprocess(self, x):
         """Load a single example using this field, tokenizing if necessary."""
         if self.sequential and isinstance(x, six.string_types):
             x = self.tokenize(x)
+        x = Pipeline(six.text_type)(x)
```

Comment:
Hmm. It wasn't before for the `lower=True` case but you're right that that probably makes more sense. Is there any reason a Py2 user would provide a tokenization function that only works on bytes? Maybe BPE? I guess in that case they should just wrap it in their own encode/decode step and have the torchtext public interface always be Unicode.